lib/opentracing/span.rb

```diff
@@ -44,7 +44,7 @@ def get_baggage_item(key)
     # @param event [String] event name for the log
```

Comment:
The `event` parameter has been removed, but the documentation remains.

Reply:
good catch!  Thanks!